pkg/scheduler/actions/preempt/preempt.go

```diff
@@ -298,11 +298,7 @@ func victimTasks(ssn *framework.Session) {
 	stmt := framework.NewStatement(ssn)
 	tasks := make([]*api.TaskInfo, 0)
 	victimTasksMap := ssn.VictimTasks(tasks)
-	victimTasks := make([]*api.TaskInfo, 0)
-	for task := range victimTasksMap {
-		victimTasks = append(victimTasks, task)
-	}
-	for _, victim := range victimTasks {
```

Comment:
@kingeasternsun Thanks for the optimization. I approve this patch.

Reply:
Ok ,Thanks.